### PR TITLE
es: Fix note blocks

### PR DESCRIPTION
**Pull request by queengooborg**
This PR fixes a bunch of note blocks throughout the Spanish locale.  These issues were a result of numerous problems with the pre-Markdown localizations, from a difference in wording choice to bad formatting.

Advertencia: hablo un poquito de español, pero no hablo mucho.  Estoy un autor del contenido ingles y una ingeniera de Yari.

### CODE REVIEW

File: files/es/learn/html/multimedia_and_embedding/video_and_audio_content/index.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Una solución nativa podría resolver mucho de esto si es implementado correctam
...
+> **Observación:** Antes de empezar, también deberías saber que hay un puñado de OVPs (proveedores de video online) como [YouTube](https://www.youtube.com/), [Dailymotion](http://www.dailymotion.com) y [Vimeo](https://vimeo.com/), y proveedores de audio como [Soundcloud](https://soundcloud.com/). Tales compañías ofrecen una conveniente fácil forma de hospedar y consumir videos, y que no tienes que preocuparte sobre el enorme ancho de banda que se consume. Los OVPs normalmente usan código prefabricado para incrustar video/audio en tus sitios web; si usas ese camino, puedes evitar algunas dificultates que discutimos en este artículo. Discutiremos este tipo de servicios un poco más en el siguiente artículo.
```
Comment (Graywolf9) [inline files/es/learn/html/multimedia_and_embedding/video_and_audio_content/index.md:50]: As `en-US`

```suggestion
> **Nota:** Antes de empezar, también deberías saber que hay un puñado de OVPs (proveedores de video online) como [YouTube](https://www.youtube.com/), [Dailymotion](http://www.dailymotion.com) y [Vimeo](https://vimeo.com/), y proveedores de audio como [Soundcloud](https://soundcloud.com/). Tales compañías ofrecen una conveniente fácil forma de hospedar y consumir videos, y que no tienes que preocuparte sobre el enorme ancho de banda que se consume. Los OVPs normalmente usan código prefabricado para incrustar video/audio en tus sitios web; si usas ese camino, puedes evitar algunas dificultades que discutimos en este artículo. Discutiremos este tipo de servicios un poco más en el siguiente artículo.
```

Comment (queengooborg) [reply files/es/learn/html/multimedia_and_embedding/video_and_audio_content/index.md:50]: Hey @Graywolf9, feel free to directly commit any localization changes like this directly to any PRs of mine!

File: files/es/web/javascript/data_structures/index.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ Infinity
...
 
-> **Nota:** **Precaución**: Aunque los operadores bit a bit se _pueden_ usar para representar múltiples valores Booleanos dentro de un solo número usando el [enmascaramiento de bits](<https://es.wikipedia.org/wiki/Máscara_(informática)>), esto generalmente se considera una mala práctica. JavaScript ofrece otros medios para representar un conjunto de valores booleanos (como un arreglo de valores booleanos o un objeto con valores booleanos asignados a propiedades con nombre). El enmascaramiento de bits también tiende a hacer que el código sea más difícil de leer, comprender y mantener.
+> **Advertencia:** Aunque los operadores bit a bit se _pueden_ usar para representar múltiples valores Booleanos dentro de un solo número usando el [enmascaramiento de bits](<https://es.wikipedia.org/wiki/Máscara_(informática)>), esto generalmente se considera una mala práctica. JavaScript ofrece otros medios para representar un conjunto de valores booleanos (como un arreglo de valores booleanos o un objeto con valores booleanos asignados a propiedades con nombre). El enmascaramiento de bits también tiende a hacer que el código sea más difícil de leer, comprender y mantener.
```
Comment (Graywolf9) [inline files/es/web/javascript/data_structures/index.md:78]: As `en-US`

```suggestion
> **Nota:** Aunque los operadores bit a bit se _pueden_ usar para representar múltiples valores Booleanos dentro de un solo número usando el [enmascaramiento de bits](<https://es.wikipedia.org/wiki/Máscara_(informática)>), esto generalmente se considera una mala práctica. JavaScript ofrece otros medios para representar un conjunto de valores booleanos (como un arreglo de valores booleanos o un objeto con valores booleanos asignados a propiedades con nombre). El enmascaramiento de bits también tiende a hacer que el código sea más difícil de leer, comprender y mantener.
```

File: files/es/web/javascript/guide/control_flow_and_error_handling/index.md [hunk 3]
```diff
@@ -136,7 +136,7 @@ Los siguientes valores se evalúan como `false` (también conocidos como valores
 
 Todos los demás valores, incluidos todos los objetos, se evalúan como `true` cuando se pasan a una declaración condicional.
 
-> **Nota:** **Precaución**: ¡No confundas los valores booleanos primitivos `true` y `false` con los valores `true` y `false` del objeto {{JSxRef("Boolean")}}!.Por ejemplo:```js
+> **Advertencia:** ¡No confundas los valores booleanos primitivos `true` y `false` con los valores `true` y `false` del objeto {{JSxRef("Boolean")}}!.Por ejemplo:```js
```
Comment (Graywolf9) [inline files/es/web/javascript/guide/control_flow_and_error_handling/index.md:139]: As `en-US`

```suggestion
> **Nota:** ¡No confundas los valores booleanos primitivos `true` y `false` con los valores `true` y `false` del objeto {{JSxRef("Boolean")}}!.Por ejemplo:```js
```

File: files/es/web/javascript/guide/functions/index.md [hunk 4]
```diff
@@ -484,7 +484,7 @@ var getCode = (function () {
 getCode(); // Devuelve el apiCode
 ```
 
-> **Nota:** **Precaución** ¡Hay una serie de trampas a tener en cuenta al usar cierres!
+> **Advertencia:** ¡Hay una serie de trampas a tener en cuenta al usar cierres!
```
Comment (Graywolf9) [inline files/es/web/javascript/guide/functions/index.md:487]: As `en-US`

```suggestion
> **Nota:** ¡Hay una serie de trampas a tener en cuenta al usar cierres!
```

File: files/es/web/javascript/guide/grammar_and_types/index.md [hunk 5]
```diff
@@ -439,7 +439,7 @@ Sin embargo, al escribir tu propio código, debes declarar explícitamente los e
 
 El tipo booleano tiene dos valores literales: `true` y `false`.
 
-> **Nota:** **Ten cuidado**: No confundas los valores booleanos primitivos `true` y `false` con los valores `true` y `false` del objeto {{JSxRef("Boolean")}}.El objeto `Boolean` es un contenedor alrededor del tipo de dato primitivo `Boolean`. Consulta {{JSxRef("Boolean")}} para obtener más información.
+> **Advertencia:** No confundas los valores booleanos primitivos `true` y `false` con los valores `true` y `false` del objeto {{JSxRef("Boolean")}}.El objeto `Boolean` es un contenedor alrededor del tipo de dato primitivo `Boolean`. Consulta {{JSxRef("Boolean")}} para obtener más información.
```
Comment (Graywolf9) [inline files/es/web/javascript/guide/grammar_and_types/index.md:442]: As `en-US`

```suggestion
> **Nota:** No confundas los valores booleanos primitivos `true` y `false` con los valores `true` y `false` del objeto {{JSxRef("Boolean")}}.El objeto `Boolean` es un contenedor alrededor del tipo de dato primitivo `Boolean`. Consulta {{JSxRef("Boolean")}} para obtener más información.
```

**Review by Graywolf9 — APPROVED** [review]
Thank you so much for this great help and your time!

**Pull request merged**